toolkits/global/packages/global-forms/view/fields/email.hbs

```diff
@@ -0,0 +1,7 @@
+{{> label}}
+{{#if error}}
+	{{> error}}
+{{/if}}
+<input type="email" class="c-forms__input c-forms__input--email"
```

Comment:
why do we need separate templates for different types of inputs? couldn't we just pass in the type that we would like to use and by default inputs are set to text

Reply:
Maybe because of `c-forms__input--email`

Reply:
unless we style it using `[type="email"]`

Reply:
@aliuk2012 Not really, because component selection is already dynamic and some inputs are actually selects, textarea, etc.

Reply:
if `<input...` was made generic then then that view template could be reused for radio/checkboxes/file and all the other various type of inputs.

Reply:
No it couldn't, because radios are together represent one field.